cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/common/StructuredTableRegistry.java

```diff
@@ -27,12 +29,6 @@
  */
 @Beta
 public interface StructuredTableRegistry {
```

Comment:
This change moves this class out of SPI module, and also there is some changes in `StructuredTableAdmin` about get spec for the table, which makes the SPI backward incompatible. I know it is no big deal since there is no other storage implementation using the SPI now. 
Just wondering in the future, for the SPI changes, should we also deprecate first and remove on a major release?

Reply:
Currently all SPI implementations are inside cdap repo, also, these APIs are marked as `@Beta`. In future, once we removed the `@Beta` annotation, then we should be careful about the changes.